tests.py

```diff
@@ -59,5 +60,36 @@ def test_get_weather(self):
             )
 
 
+# Checking for Twitter API response
+class TwitterTests(unittest.TestCase):
+    """We'll test the process of creating an object
+    and putting it into the database"""
+
+    def test_twitter_api(self):
+        mock_reponse_api = MagicMock()
+        mock_reponse_api.return_value = [
+            {
+                "trends": [
+                    {"name": "a"},
+                    {"name": "b"},
+                    {"name": "c"},
+                    {"name": "d"},
+                    {"name": "e"},
+                ]
+            }
+        ]
+        mock_response_auth = MagicMock()
+
+        mock_response_auth.return_value = "92879e737e983748308748374987y489y8gf8wgef8ub"
+
+        with patch("twitter.tweepy.OAuthHandler") as mock_auth:
+            with patch("twitter.tweepy.API.get_place_trends") as mock_api:
+                mock_auth.return_value = mock_response_auth
+
+                mock_api.return_value = mock_reponse_api.return_value
+
+                self.assertEqual(get_trends(), ["a", "b", "c", "d", "e"])
```

Comment:
If our test is able to retrieve this list from the original array/list, that means it'll be able to retrieve the trending topics when the API call is made.